include/sheep.hrl

```diff
@@ -2,20 +2,20 @@
 
 -record(sheep_request, {
     method:: binary(),
-    headers = [] :: list(),
-    query = #{} :: map(),
-    bindings = #{} :: map(),
-    body = <<>> :: map() | [map()] | binary()
+    headers = #{} :: maps:map(),
```

Comment:
Is case with duplicated headers supported after switching record field type to map?
E.g.: https://stackoverflow.com/questions/4371328/are-duplicate-http-response-headers-acceptable